content/ngf/upgrading-ngf.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Upgrade NGINX Gateway Fabric to version 2.x
+weight: 400
+toc: true
+type: how-to
+product: NGF
+docs: DOCS-0000
+---
+
+This guide provides step-by-step instructions for upgrading NGINX Gateway Fabric from version 1.x to 2.x, highlighting key architectural changes, expected downtime, and important considerations for custom resource definitions (CRDs).
+
+
+### Upgrade from v1.x to v2.x
+
+To upgrade NGINX Gateway Fabric from version 1.x to the new architecture in version 2.x, you must uninstall the existing NGINX Gateway Fabric CRDs and deployment, and perform a fresh installation. This will cause brief downtime during the upgrade process.
+
+{{<note>}} You do not need to uninstall the Gateway API CRDs during the upgrade. These resources are compatible with the new NGINX Gateway Fabric version. {{</note>}}
+
+#### Uninstall NGINX Gateway Fabric v1.x
+
+To remove the previous version 1.x of NGINX Gateway Fabric, follow these steps:
+
+First, run the following command to uninstall NGINX Gateway Fabric from the `nginx-gateway` namespace, and update `ngf` to your release name if it is different:
+
+```shell
+helm uninstall ngf -n nginx-gateway 
+```
+
+Afterwards, remove CRDs associated with NGINX Gateway Fabric version 1.x with the following command:
+
+```shell
+kubectl delete -f https://raw.githubusercontent.com/nginx/nginx-gateway-fabric/v{{< version-ngf >}}/deploy/crds.yaml
```

Comment:
Just realized, this needs to be hardcoded to 1.6.2, because otherwise when we update the verison, it'll say 2.0. But the installation one below can stay as is.